keyboards/mokey/mokey12x2/info.json

```diff
@@ -0,0 +1,56 @@
+{
+    "keyboard_name": "Mokey12x2",
+    "manufacturer": "Mokey",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x6653",
+        "pid": "0x012A",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["A6", "A5", "A4", "A3", "B9"],
+        "rows": ["A2", "A1", "A0", "NO_PIN"]
```

Comment:
Padding with `NO_PIN` here is incorrect, given nothing will then trigger the matrix locations.